pepc.html

```diff
@@ -8,14 +8,48 @@
     <link rel="shortcut icon" href="favicon.ico" />
     <link rel="apple-touch-icon" href="app-icon.png" />
     <link rel="stylesheet" href="style.css" />
+    <script src="pepc.js"></script>
   </head>
   <body>  
     <div class="before"></div>
     <div class="content">      
-      <div><permission id="camera" type="camera"></permission></div>
-      <div><permission id="microphone" type="microphone"></permission></div>
-      <div><permission id="camera-microphone" type="camera microphone"></permission></div>
-      <div><permission id="geolocation" type="geolocation"></permission></div>
+      <span class="permission-element-box">
+        <div><permission id="geolocation" type="geolocation"></permission></div>

```

Comment:
Could we add some sort of placeholder for clarity?
```suggestion
        <div><permission id="geolocation" type="geolocation">(This <code>&lt;permission&gt;</code> element is inactive.)</permission></div>

```

Reply:
Note this type of placeholder syntax is currenlty only supported on Chrome canary.